web-common/src/features/file-explorer/ForceDeleteConfirmation.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import { Button } from "@rilldata/web-common/components/button";
+  import Dialog from "@rilldata/web-common/components/dialog/Dialog.svelte";
```

Comment:
(Nice-to-have, but by no means a blocker): it looks like ShadCN-Svelte has a specific [Alert Dialog component](https://www.shadcn-svelte.com/docs/components/alert-dialog) for these use cases.

Reply:
Good idea. Seems like it has some dependancy on button. Lets add it in a separate PR and move all alerts to use this.